plugin/runtime/host.py

```
@@ -159,54 +416,114 @@ def _run_timer_interval(fn, interval_seconds: int, fn_name: str, stop_event: thr
                     
                     if asyncio.iscoroutinefunction(method):
-                        res = asyncio.run(method(**args))
+                        logger.debug("[Plugin Process] Method is async, running in thread to avoid blocking command loop")
+                        # 关键修复：在独立线程中运行异步方法，避免阻塞命令循环
+                        # 这样命令循环可以继续处理其他命令（包括响应命令）
+                        result_container = {"result": None, "exception": None, "done": False}
+                        event = threading.Event()
+                        
+                        def run_async(method=method, args=args, result_container=result_container, event=event):
+                            try:
+                                with ctx._handler_scope(f"plugin_entry.{entry_id}"):
```

Ruff (0.14.10)

Function definition does not bind loop variable `entry_id`

(B023)